source/predicates/predicate.ts

```diff
@@ -4,6 +4,14 @@ import {not} from '../operators/not';
 import {BasePredicate, testSymbol} from './base-predicate';
 import {Main} from '..';
 
+/**
+Function executed when the provided validation fails.
+The first argument provided to the function is the provided `value` for the property,
+the second argument is the optional `label` for the property.
+The returned value will be the error message.
```

Comment:
JSDoc has special syntax for documenting parameters and return value.

Reply:
Used the `@param` and `@returns` syntax